pkg/storaged/content-views.jsx

```diff
@@ -152,18 +152,32 @@ function create_tabs(client, target, is_partition) {
         if (!crypto)
             return;
 
-        dialog.open({ Title: _("Unlock"),
-                      Fields: [
-                          { PassInput: "passphrase",
-                            Title: _("Passphrase")
-                          }
-                      ],
-                      Action: {
-                          Title: _("Unlock"),
-                          action: function (vals) {
-                              return crypto.Unlock(vals.passphrase, {});
+        /* If there is a stored passphrase, the Unlock method will
+         * use it unconditionally.  So we don't ask for one in
+         * that case.
+         */
```

Comment:
I would suggest documenting the format of items in the comment here? Or perhaps link to the documentation: https://people.freedesktop.org/~david/udisks2-20111102/gdbus-org.freedesktop.UDisks2.Block.html#gdbus-method-org-freedesktop-UDisks2-Block.GetSecretConfiguration